change/@fluentui-react-dialog-c4b3b805-3dcc-4caa-9c9e-ceb9a1580a1a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29391&commitId=f2dbb2523669ad3e4a9d3c4d46f7d5dd2ba84706&env=prod&clientType=FLUENTUI)


  <details><summary><b>Dialog</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Fluid Actions End.chromium.png|702290| Changed|
|Dialog.actions position start - RTL.chromium.png|701405| Changed|
|Dialog.actions position start & position end.chromium.png|701735| Changed|
|Dialog.actions position start.chromium.png|701406| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|765105| Changed|
|Dialog.actions position start - High Contrast.chromium.png|774133| Changed|
|Dialog.default - High Contrast.chromium.png|774133| Changed|
|Dialog.default - Dark Mode.chromium.png|765106| Changed|
|Dialog.alert - High Contrast.chromium.png|774133| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|773537| Changed|
|Dialog.alert - RTL.chromium.png|701406| Changed|
|Dialog.alert - Dark Mode.chromium.png|765106| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|764767| Changed|
|Dialog.alert.chromium.png|701406| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|701735| Changed|
|Dialog.Fluid Actions Start.chromium.png|702289| Changed|
|Dialog.nested - Dark Mode.chromium.png|761459| Changed|
|Dialog.nested.chromium.png|707347| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|754078| Changed|
|Dialog.no actions - High Contrast.chromium.png|763546| Changed|
|Dialog.nested - High Contrast.chromium.png|768365| Changed|
|Dialog.default - RTL.chromium.png|701406| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|748543| Changed|
|Dialog.no actions - RTL.chromium.png|708481| Changed|
|Dialog.no title & no actions - RTL.chromium.png|714578| Changed|
|Dialog.default.chromium.png|701406| Changed|
|Dialog.no actions.chromium.png|708481| Changed|
|Dialog.no title - Dark Mode.chromium.png|758171| Changed|
|Dialog.nested - RTL.chromium.png|707347| Changed|
|Dialog.no actions - Dark Mode.chromium.png|755478| Changed|
|Dialog.no title & no actions.chromium.png|714578| Changed|
|Dialog.no title - High Contrast.chromium.png|764666| Changed|
|Dialog.no title.chromium.png|707504| Changed|
|Dialog.no title - RTL.chromium.png|707503| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|988079| Changed|
|Dialog.non-modal - High Contrast.chromium.png|107396| Changed|
|Dialog.non-modal.chromium.png|9252| Changed|
|Dialog.non-modal - RTL.chromium.png|9252| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|94667| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|916037| Changed|
|Dialog.title custom action.chromium.png|707882| Changed|
|Dialog.title custom action - High Contrast.chromium.png|764570| Changed|
|Dialog.scroll long content - RTL.chromium.png|559983| Changed|
|Dialog.title custom action - RTL.chromium.png|707881| Changed|
|Dialog.scroll long content.chromium.png|559983| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|756209| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>